### DISCUSSION TITLE
v1.4.6.1

### DISCUSSION THREAD

**Discussion by buresdv** [buresdv/Cork#423] [Announcements]
## What's Changed
* Navigation adjustments by @buresdv in https://github.com/buresdv/Cork/pull/420
* Simplify `Mise` installation in README by @rakleed in https://github.com/buresdv/Cork/pull/403
* Adding Portuguese translation by @DGoiana in https://github.com/buresdv/Cork/pull/422

## New Contributors
* @rakleed made their first contribution in https://github.com/buresdv/Cork/pull/403
* @DGoiana made their first contribution in https://github.com/buresdv/Cork/pull/422

**Full Changelog**: https://github.com/buresdv/Cork/compare/v1.4.6...v1.4.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/buresdv/Cork/releases/tag/v1.4.6.1'>v1.4.6.1</a>.</em>

**Comment by insec404** [comment]
any version of this for macOS Monterey?

**Reply by buresdv** [reply]
No, there's no version for Monterey
